js/github.js

```diff
@@ -12,7 +12,10 @@ const doesGitHubCookieExist = () => {
 const updateGitHubLink = () => {
   const allLoginBtns = document.querySelectorAll('.btn-login');
   allLoginBtns.forEach((btn) => {
-    btn.setAttribute('href', 'https://github.realdevsquad.com');
+    btn.setAttribute(
```

Comment:
This doesn't look like the right place for making the override.

Could you please take care of this, where the switch between the real button and fake button takes place?

Reply:
Sure will fix it .

Reply:
I have updated the code and removed the conditional button switching part, if it's fine; please approve the PR. @ankushdharkar